components/GoalsAndMission/GoalsAndMission.js

```diff
@@ -0,0 +1,19 @@
+import { Container, Row, Col } from "../bootstrap"
+import AboutPagesCard from "../AboutPagesCard/AboutPagesCard"
+import GoalsAndMissionCardContent from "../OurGoalsAndMissionCardContent/OurGoalsCardContent"
+import styles from "../GoalsAndMission/GoalsAndMission.module.css"
+
+const GoalsAndMission = () => {
+  return (
+    <Container fluid className={styles.container}>
```

Comment:
Let's remove `fluid` here so the content has a max-width, especially for large screens

Reply:
removed